scripts/module/teleporter.nut

```diff
@@ -13,7 +13,6 @@ teleporters <- {
   "y": Vector( 557.677, -886.057, 432.612 ) };
 
 function teleport(player, location) {
-  local location = location.tolower();
```

Comment:
The teleporter function should only accept strings as second argument.

Please check for condition in `onPlayerCommand` and warn player about bad syntax.